### PR TITLE
Added additional stubs to stdole and adodb docs

### PR DESCRIPTION
**Pull request by John-Hart**
Add stubs for some adodb.dll APIs
Added a few more stubs to the stdole.dll doc

### CODE REVIEW

File: docs/framework/additional-apis/adodb.connection.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  - "Assembly"
+---
+# Connection Interface
+
+```csharp
+[GuidAttribute("00000550-0000-0010-8000-00AA006D2EA4")]
+public interface Connection : _Connection, 
+	ConnectionEvents_Event
+```
+## Requirements
```
Comment (mairaw) [inline docs/framework/additional-apis/adodb.connection.md]:
```suggestion

## Requirements
```

File: docs/framework/additional-apis/adodb.connection.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: Connection Interface (adodb)
+ms.date: 023/16/2019
+topic_type:
+  - "apiref"
+
```
Comment (mairaw) [inline docs/framework/additional-apis/adodb.connection.md]:
```suggestion
api_name:
  - "adodb.Connection"
```

File: docs/framework/additional-apis/adodb.eventreasonenum.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+title: EventReason Enumeration (adodb)
+ms.date: 023/16/2019
+topic_type:
+  - "apiref"
+
```
Comment (mairaw) [inline docs/framework/additional-apis/adodb.eventreasonenum.md]:
```suggestion
api_name:
  - "adodb.EventReason"
```

File: docs/framework/additional-apis/adodb.eventreasonenum.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  - "Assembly"
+---
+# EventReason Enumeration
+
+```csharp
+[GuidAttribute("00000531-0000-0010-8000-00AA006D2EA4")]
+public enum EventReasonEnum
+```
+## Members
+| Member name  | Description  |
```
Comment (mairaw) [inline docs/framework/additional-apis/adodb.eventreasonenum.md]:
```suggestion

| Member name  | Description  |
```

File: docs/framework/additional-apis/adodb.eventstatusenum.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+---
+title: EventStatus Enumeration (adodb)
+ms.date: 023/16/2019
+topic_type:
+  - "apiref"
+
```
Comment (mairaw) [inline docs/framework/additional-apis/adodb.eventstatusenum.md]:
```suggestion
api_name:
  - "adodb.EventStatus"
```

File: docs/framework/additional-apis/adodb.eventstatusenum.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+  - "Assembly"
+---
+# EventStatus Enumeration
+
+```csharp
+[GuidAttribute("00000530-0000-0010-8000-00AA006D2EA4")]
+public enum EventStatusEnum
+```
+## Members
+| Member name  | Description  |
```
Comment (mairaw) [inline docs/framework/additional-apis/adodb.eventstatusenum.md]:
```suggestion

| Member name  | Description  |
```

File: docs/framework/additional-apis/stdole.dispparams.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
+---
+title: DISPPARAMS Structure(stdole)
+ms.date: 023/16/2019
+topic_type:
+  - "apiref"
+
```
Comment (mairaw) [inline docs/framework/additional-apis/stdole.dispparams.md]:
```suggestion
api_name:
  - "stdole.DISPPARAMS"
```

File: docs/framework/additional-apis/stdole.dispparams.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
+---
+title: DISPPARAMS Structure(stdole)
```
Comment (mairaw) [inline docs/framework/additional-apis/stdole.dispparams.md]:
```suggestion
title: DISPPARAMS Structure (stdole)
```

File: docs/framework/additional-apis/stdole.dispparams.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+ms.date: 023/16/2019
```
Comment (mairaw) [inline docs/framework/additional-apis/stdole.dispparams.md]:
```suggestion
ms.date: 03/16/2019
```

File: docs/framework/additional-apis/adodb.connection.md [hunk 10]
```diff
@@ -0,0 +1,23 @@
+---
+title: Connection Interface (adodb)
+ms.date: 023/16/2019
```
Comment (mairaw) [inline docs/framework/additional-apis/adodb.connection.md]:
```suggestion
ms.date: 03/28/2019
```

File: docs/framework/additional-apis/adodb.eventreasonenum.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
+---
+title: EventReason Enumeration (adodb)
+ms.date: 023/16/2019
```
Comment (mairaw) [inline docs/framework/additional-apis/adodb.eventreasonenum.md]:
```suggestion
ms.date: 03/28/2019
```

File: docs/framework/additional-apis/adodb.eventstatusenum.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
+---
+title: EventStatus Enumeration (adodb)
+ms.date: 023/16/2019
```
Comment (mairaw) [inline docs/framework/additional-apis/adodb.eventstatusenum.md]:
```suggestion
ms.date: 03/28/2019
```

File: docs/framework/additional-apis/stdole.dispparams.md [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+  - "stdole.dll"
+api_type:
+  - "Assembly"
+---
+# DISPPARAMS Structure
+
+```csharp
+public struct DISPPARAMS
+```
+## Requirements
```
Comment (mairaw) [inline docs/framework/additional-apis/stdole.dispparams.md]:
```suggestion

## Requirements
```

File: docs/framework/additional-apis/stdole.excepinfo.md [hunk 14]
```diff
@@ -0,0 +1,21 @@
+---
+title: EXCEPINFO Structure (stdole)
+ms.date: 023/16/2019
```
Comment (mairaw) [inline docs/framework/additional-apis/stdole.excepinfo.md]:
```suggestion
ms.date: 03/28/2019
```

File: docs/framework/additional-apis/stdole.excepinfo.md [hunk 15]
```diff
@@ -0,0 +1,21 @@
...
+topic_type:
+  - "apiref"
+
```
Comment (mairaw) [inline docs/framework/additional-apis/stdole.excepinfo.md]:
```suggestion
api_name:
  - "stdole.EXCEPINFO"
```

File: docs/framework/additional-apis/stdole.excepinfo.md [hunk 16]
```diff
@@ -0,0 +1,21 @@
...
+  - "stdole.dll"
+api_type:
+  - "Assembly"
+---
+# EXCEPINFO Structure
+
+```csharp
+public struct EXCEPINFO
+```
+## Requirements
```
Comment (mairaw) [inline docs/framework/additional-apis/stdole.excepinfo.md]:
```suggestion

## Requirements
```

File: docs/framework/additional-apis/stdole.ifontdisp.md [hunk 17]
```diff
@@ -0,0 +1,23 @@
+---
+title: IFontDisp Interface (stdole)
+ms.date: 023/16/2019
```
Comment (mairaw) [inline docs/framework/additional-apis/stdole.ifontdisp.md]:
```suggestion
ms.date: 03/28/2019
```

File: docs/framework/additional-apis/stdole.ifontdisp.md [hunk 18]
```diff
@@ -0,0 +1,23 @@
...
+topic_type:
+  - "apiref"
+
```
Comment (mairaw) [inline docs/framework/additional-apis/stdole.ifontdisp.md]:
```suggestion
api_name:
  - "stdole.IFontDisp"
```

File: docs/framework/additional-apis/stdole.ifontdisp.md [hunk 19]
```diff
@@ -0,0 +1,23 @@
...
+  - "Assembly"
+---
+# IPictureFont Interface
+
+```csharp
+[GuidAttribute("BEF6E003-A874-101A-8BBA-00AA00300CAB")]
+[InterfaceTypeAttribute()]
+public interface IFontDisp
+```
+## Requirements
```
Comment (mairaw) [inline docs/framework/additional-apis/stdole.ifontdisp.md]:
```suggestion

## Requirements
```

File: docs/framework/additional-apis/stdole.ipicturedisp.md [hunk 20]
```diff
@@ -0,0 +1,23 @@
+---
+title: IPictureDisp Interface (stdole)
+ms.date: 023/16/2019
```
Comment (mairaw) [inline docs/framework/additional-apis/stdole.ipicturedisp.md]:
```suggestion
ms.date: 03/28/2019
```

File: docs/framework/additional-apis/stdole.ipicturedisp.md [hunk 21]
```diff
@@ -0,0 +1,23 @@
...
+topic_type:
+  - "apiref"
+
```
Comment (mairaw) [inline docs/framework/additional-apis/stdole.ipicturedisp.md]:
```suggestion
api_name:
  - "stdole.IPictureDisp"
```

File: docs/framework/additional-apis/stdole.ipicturedisp.md [hunk 22]
```diff
@@ -0,0 +1,23 @@
...
+  - "Assembly"
+---
+# IPictureDisp Interface
+
+```csharp
+[InterfaceTypeAttribute()]
+[GuidAttribute("7BF80981-BF32-101A-8BBB-00AA00300CAB")]
+public interface IPictureDisp
+```
+## Requirements
```
Comment (mairaw) [inline docs/framework/additional-apis/stdole.ipicturedisp.md]:
```suggestion

## Requirements
```

File: docs/framework/additional-apis/stdole.stdfont.md [hunk 23]
```diff
@@ -0,0 +1,23 @@
+---
+title: StdFont Interface (stdole)
+ms.date: 023/16/2019
+topic_type:
+  - "apiref"
+
```
Comment (mairaw) [inline docs/framework/additional-apis/stdole.stdfont.md]:
```suggestion
```suggestion
api_name:
  - "stdole.StdFont"
```
```

File: docs/framework/additional-apis/stdole.stdfont.md [hunk 24]
```diff
@@ -0,0 +1,23 @@
+---
+title: StdFont Interface (stdole)
+ms.date: 023/16/2019
```
Comment (mairaw) [inline docs/framework/additional-apis/stdole.stdfont.md]:
```suggestion
ms.date: 03/28/2019
```

File: docs/framework/additional-apis/stdole.stdfont.md [hunk 25]
```diff
@@ -0,0 +1,23 @@
...
+  - "Assembly"
+---
+# StdFont Interface
+
+```csharp
+[GuidAttribute("BEF6E003-A874-101A-8BBA-00AA00300CAB")]
+public interface StdFont : Font, 
+	FontEvents_Event
+```
+## Requirements
```
Comment (mairaw) [inline docs/framework/additional-apis/stdole.stdfont.md]:
```suggestion

## Requirements
```

File: docs/framework/additional-apis/stdole.stdpicture.md [hunk 26]
```diff
@@ -0,0 +1,22 @@
+---
+title: StdPicture Interface (stdole)
+ms.date: 023/16/2019
```
Comment (mairaw) [inline docs/framework/additional-apis/stdole.stdpicture.md]:
```suggestion
ms.date: 03/28/2019
```

File: docs/framework/additional-apis/stdole.stdpicture.md [hunk 27]
```diff
@@ -0,0 +1,22 @@
...
+topic_type:
+  - "apiref"
+
```
Comment (mairaw) [inline docs/framework/additional-apis/stdole.stdpicture.md]:
```suggestion
api_name:
  - "stdole.StdPicture"
```

File: docs/framework/additional-apis/stdole.stdpicture.md [hunk 28]
```diff
@@ -0,0 +1,22 @@
...
+api_type:
+  - "Assembly"
+---
+# StdPicture Interface
+
+```csharp
+[GuidAttribute("7BF80981-BF32-101A-8BBB-00AA00300CAB")]
+public interface StdPicture : Picture
+```
+## Requirements
```
Comment (mairaw) [inline docs/framework/additional-apis/stdole.stdpicture.md]:
```suggestion

## Requirements
```

File: docs/framework/additional-apis/toc.yml [hunk 29]
```diff
@@ -82,8 +82,28 @@
...
     items:
+    - name: DISPPARAMS Structure
+      href: stdole.dispparams.md
+    - name: EXCEPINFO Structure
+      href: stdole.excepinfo.md
+    - name: IFontDisp Interface
+      href: stdole.ifontdisp.md
+    - name: IPictureDisp Interface
+      href: stdole.ipicturedisp.md
+    - name: StdFont Interface
```
Comment (mairaw) [inline docs/framework/additional-apis/toc.yml]: Can you add these in alphabetical order?

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @John-Hart and apologies on the delay for getting this reviewed. Content looks good but there were a few things to fix: bad ms.date format, missing api name for apiscan, and some other small fixes. You can batch most of my suggestions via the Files changed tab.

File: docs/framework/additional-apis/toc.yml (hunk 29 above)
Comment (John-Hart) [reply docs/framework/additional-apis/toc.yml]: @mairaw  Sure, I thought it was better to be by type (i.e. Structures, Intefaces, Properties) and alphabetical within each type

Comment (John-Hart) [reply docs/framework/additional-apis/toc.yml]: I've sorted the stdole entries alphabetical

**Review by mairaw — APPROVED** [review]
LGTM

**Pull request merged**